src/Notification.vala

```diff
@@ -35,16 +34,25 @@ public class Notifications.Notification : GLib.Object {
     public GLib.Icon? badge_icon { get; set; default = null; }
     public MaskedImage? image { get; set; default = null; }
 
+    public string default_action_name { get; set; }
+    public Variant default_action_target { get; set; }
```

Comment:
This doesn't seem to get set?

Reply:
not used now, as Fdo Interface doesn't have action parameters, we can deserialize the action string, however i worry that we can cause issues to non-glib applications, since a action named `acton::target` would end as `action('target')` when the action ended activated.